bookshop/.env.example

```diff
@@ -0,0 +1 @@
+API_URL=http://api.local:8081/api/
```

Comment:
use url that compatible with docker container

Reply:
add empty line

Reply:
ok